src/AutoRest.CSharp/Common/Utilities/StringExtensions.cs

```diff
@@ -25,6 +25,8 @@ static StringExtensions()
         public static bool IsNullOrEmpty(this string? text) => String.IsNullOrEmpty(text);
         public static bool IsNullOrWhiteSpace(this string? text) => String.IsNullOrWhiteSpace(text);
 
+        private static bool IsWordSeparator(char c) => !SyntaxFacts.IsIdentifierPartCharacter(c) || c == '_';
+
         [return: NotNullIfNotNull("name")]
         public static string ToCleanName(this string name, bool camelCase = true)
```

Comment:
is this boolean `isCamelCase` or `wantCamelCase`, can we rename it accordingly?

Reply:
Make sense. I will rename this to `WantCamelCase`